### ISSUE TITLE
Ansible Operator Proxy - Caching Issue

### ISSUE THREAD

**Issue by nhariha1** [operator-framework/operator-sdk#4519]
#### What did you do?

I'm developing an ansible based operator and all tasks that hit the API server produces several "Skipping Cache lookup" on multiple API Paths. 

The task is as simple as getting all the Services in a given namespace: 

```
- name: get service names
  community.kubernetes.k8s_info:
    api_version: v1
    kind: Service
    namespace: "{{ ansible_operator_meta.namespace }}"
  register: services_res
```

#### What did you expect to see?

I expected the usual Ansible task's output lines in the operator pod logs. 

#### What did you see instead? Under which circumstances?

There were over 50 lines of Skip Cache Lookup in the logs. 

```
2021-02-12T14:16:35.554-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/version","Verb":"get","APIPrefix":"","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:35.600-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/version/openshift","Verb":"get","APIPrefix":"","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:35.645-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis","Verb":"get","APIPrefix":"","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:35.739-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/api/v1","Verb":"get","APIPrefix":"","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:35.788-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/apiregistration.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:35.834-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/apps/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:35.879-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/events.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:35.924-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/authentication.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:35.973-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/authorization.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.019-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/autoscaling/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.067-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/batch/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.112-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/certificates.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.158-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/networking.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.214-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/rbac.authorization.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.262-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/storage.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.307-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/admissionregistration.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.351-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/apiextensions.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.400-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/scheduling.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.448-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/coordination.k8s.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.492-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/apps.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.542-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/authorization.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.592-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/build.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.638-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/image.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.686-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/oauth.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.741-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/project.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.791-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/quota.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.841-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/route.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.891-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/security.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:36.957-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/template.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.013-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/user.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.085-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/packages.operators.coreos.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.135-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/config.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.179-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/operator.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.227-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/autoscaling.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.273-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/ceph.rook.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.321-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/cloudcredential.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.366-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/clusterhealth.ibm.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.408-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/console.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.457-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/foundation.ibm.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.503-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/hbstrider.ibm.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.549-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/ibmcpcs.ibm.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.600-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/imageregistry.operator.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.655-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/ingress.operator.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.699-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/k8s.cni.cncf.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.745-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/machineconfiguration.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.788-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/metering.ibm.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.836-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/monitoring.coreos.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.881-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/monitoringcontroller.cloud.ibm.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:37.925-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/network.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:38.002-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/network.operator.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:38.048-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/oidc.security.ibm.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:38.096-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/operator.ibm.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:38.141-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/operators.coreos.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:38.197-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/redis.databases.cloud.ibm.com/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:38.239-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/samples.operator.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:38.281-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/security.internal.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:38.328-0500	INFO	proxy	Skipping cache lookup	{"resource": {"IsResourceRequest":false,"Path":"/apis/tuned.openshift.io/v1","Verb":"get","APIPrefix":"apis","APIGroup":"","APIVersion":"","Namespace":"","Resource":"","Subresource":"","Name":"","Parts":null}}
2021-02-12T14:16:38.501-0500	INFO	proxy	Read object from cache	{"resource": {"IsResourceRequest":true,"Path":"/api/v1/namespaces/hb-test/services","Verb":"list","APIPrefix":"api","APIGroup":"","APIVersion":"v1","Namespace":"hb-test","Resource":"services","Subresource":"","Name":"","Parts":["services"]}}
```

#### Environment

**Operator type:**

/language ansible

**Kubernetes cluster type:**

The type of cluster used for testing/deployment,  "OpenShift"

`$ operator-sdk version`

quay.io/operator-framework/ansible-operator:v1.3.0

`$ go version` (if language is Go)

`$ kubectl version`

#### Additional context

Conversation in this thread - 
https://groups.google.com/d/msgid/operator-framework/12a9a578-99a1-4efc-a2ae-7ada40f2c73an%40googlegroups.com.

**Comment by jmrodri** [comment]
Fixed by PR #4511 

Before this change:

```
$ grep "Skipping cache" good.log | wc -l
48
```

With the above PR

```
$ grep "Skipping cache" 20210216-bad.log  | wc -l
0
```

**Comment by jmrodri** [comment]
Thanks @kandaaaaa for fixing this issue before it was filed. Great work!
